src/main/java/com/scalar/cassy/transferer/AzureBlobFileUploader.java

```diff
@@ -66,10 +67,17 @@ public void upload(List<Path> files, BackupConfig config) {
       } else {
         logger.info(filePath + " has been already uploaded.");
       }
+      if (toBeUploaded.size() % ASYNC_FILE_UPLOAD_LIMIT == 0) {
```

Comment:
Why are we using a modulo operator (%) here? Is it okay that we enter this `if` block in case of `toBeUploaded.size() == 0`?

It looks like just `if (toBeUploaded.size() == ASYNC_FILE_UPLOAD_LIMIT)` or `if (toBeUploaded.size() >= ASYNC_FILE_UPLOAD_LIMIT)` is fine?

Reply:
@brfrn169 san,
Thank you for your suggestions.
Suggestions (`if (toBeUploaded.size() >= ASYNC_FILE_UPLOAD_LIMIT)`) updated accordingly.